hr_skill/models/hr_skill.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#    Copyright (C) 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp import fields, models
+
+
+class Skill(models.Model):
+    _name = 'hr.skill'
```

Comment:
Falling appart the comments about the form details +1

2015-08-05 5:23 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> In hr_skill/models/hr_skill.py
> <https://github.com/OCA/hr/pull/124#discussion_r36287819>:
>
> > +#
> > +#    This program is distributed in the hope that it will be useful,
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
> > +#    GNU Affero General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU Affero General Public License
> > +#    along with this program. If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +###############################################################################
> > +
> > +from openerp import fields, models
> > +
> > +
> > +class Skill(models.Model):
> > +    _name = 'hr.skill'
>
> Add _parent_store = True for enabling the operator child_of
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/hr/pull/124/files#r36287819>.
>



-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>